app/src/encounters/encountersActions.js

```diff
@@ -77,8 +77,9 @@ export function setEncountersInfo(seriesgroup, tilesetId) {
       const workspaceLayers = getState().layers.workspaceLayers;
 
       encounterInfo.vessels.forEach((vessel) => {
-        const workspaceLayer = workspaceLayers.find(layer => layer.tilesetId === vessel.tilesetId);
+        const workspaceLayer = workspaceLayers.find(l => l.tilesetId === vessel.tilesetId || l.id === vessel.tilesetId);
```

Comment:
@nerik I  don't like variable names that are not self_descriptive and 1 character of length. Could you please change the name of the variable? Besides that the change looks good.

Reply:
@enriquetuya @nerik while I would generally agree with you @enriquetuya, in lambda functions it's usually ok. Usually, when you have a single-argument lambda, it's common to use `it` as the argument name, as the lambda is already descriptive enough.

Reply:
@andres-arana I think I fixed while you were writing your comment :p